### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by bwohlberg** [lanl/scico#139]
The [installation instructions](https://scico.readthedocs.io/en/latest/install.html) need to be updated to reflect the new recommended way of installing `jaxlib`. Updating of the `astra-toolbox` installation instructions should also be considered since the [conda version](https://anaconda.org/astra-toolbox/astra-toolbox) (2.0.0) is now substantially more recent than the [pip version](https://pypi.org/project/astra-toolbox/) (1.8b5).

Also note https://scico.readthedocs.io/en/latest/contributing.html#scico-dev-contributing does not mention `dev_requirements.txt` and it probably should.

**Comment by smajee** [comment]
I would also add `cached_property` in `example_requirements` since the PyPi version of xdesign doesn't seem to include this and I always get a missing `cached_property` error whenever I do a fresh installation.

**Comment by bwohlberg** [comment]
It would be best to also report a bug at [the xdesign repo](https://github.com/tomography/xdesign). I see that `cached-property` is listed in their `requirements.txt` file, so the problem may be due to the absence of any `install_requires` section in their `setup.py`.

**Comment by smajee** [comment]
Good idea. Added an issue in the xdesign repo.

**Comment by Michael-T-McCann** [comment]
Specifically what kind of install has incorrect instructions? From pip or from github? GPU or CPU?

**Comment by bwohlberg** [comment]
For jaxlib, we currently recommend something like
```
pip install --upgrade "jaxlib==0.1.70+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
but it seems that
```
pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
is now a cleaner way of installing, no? Or does this only upgrade jax, and not the jaxlib dependency?

For astra-toolbox, we currently recommend
```
conda install -c astra-toolbox/label/dev astra-toolbox
```
but the version available via pip is now a lot more recent.

**Comment by Michael-T-McCann** [comment]
The cleaner way of installing jax will (I believe) give you the latest version of jax and jaxlib, not the pinned one needed by SCICO.

For astra, we currently recommend the _conda_ version and _conda_ version is more recent. This is fine, right?

**Comment by bwohlberg** [comment]
You're right. I had just assumed that we were recommending pip installation since that's the default for everything else.

OK, then I guess nothing to do here other than to add a note on `xdesign` requirements as pointed out by @smajee.

**Comment by smajee** [comment]
We would also have to modify the GPU install command to reflect the newer version of jax in scico : `pip install --upgrade "jaxlib==0.1.75+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html` (0.1.75 from 0.1.70

**Comment by smajee** [comment]
It seems that we do recommend installation of the `example_requirements` via pip in [docs/source/contributing.rst](https://scico.readthedocs.io/en/latest/contributing.html#scico-dev-contributing). This includes the `astra-toolbox` which should be installed via conda.

The instructions in [https://scico.readthedocs.io/en/latest/install.html#additional-dependencies](https://scico.readthedocs.io/en/latest/install.html#additional-dependencies) seem to be correct.

Should I add a note in  [docs/source/contributing.rst](https://scico.readthedocs.io/en/latest/contributing.html#scico-dev-contributing) that astra should be installed via the conda command and the rest via pip using `example_requirements.txt`?

**Comment by bwohlberg** [comment]
For now, let's just add the note to `docs/source/contributing.rst`.

When possible, though, we should discuss
* trying to avoid the partial duplication of installation instructions between the main docs and contributing docs, and
*  whether it's worth implementing the splitting of the pip requirements files or using a conda `environment.yml` file.

**Comment by smajee** [comment]
Added a note in `docs/source/contributing.rst` for now. 

There seem to be more duplications regarding building of docs.  We should discuss how to consolidate these and avoid duplications. My inclination is to:
1. Move docs build instructions to the contributing/developer section since it is unlikely that normal users will do this
2. Moving the example specific dependency installation to the separate example section and a link in the installation section that points to this

**Comment by bwohlberg** [comment]
Agreed.

**Comment by smajee** [comment]
Made the changes in pull request #154

**Comment by smajee** [comment]
Fixed in pull request #154
